setup.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
+from setuptools import find_packages
+from distutils.core import setup
+
+setup(
+	name='sphero_sdk',
+	version='0.3.4.post2',  # TODO: unify with __version__.py
```

Comment:
The `post2` is here because I uploaded 2 broken versions, and unfortunately for security reasons pypi does not allow you to overwrite a version, even if you delete it first.

I would've just set this to `version=__version__` and added a line above `from sphero_sdk.__version__ import __version__`, but because `sphero_sdk/__init__.py` imports the whole world, that would make all of the runtime dependencies of the SDK to be installed before setup.py could run.